backend/fleet_management/api.py

```diff
@@ -1,11 +1,33 @@
-from rest_framework import views
+from django_filters.rest_framework import DjangoFilterBackend
+from rest_framework import generics, filters, views
 from rest_framework.response import Response
-
-from .serializers import UserSerializer
+from .serializers import UserSerializer, RouteSerializer
+from .models import Route
+from rest_framework.permissions import IsAuthenticated
 
 
 class CurrentUserRetrieveView(views.APIView):
     def get(self, request):
         return Response(
             UserSerializer(request.user).data
         )
+
+
+class RouteListView(generics.ListAPIView):
+    authentication_classes = (IsAuthenticated,)
+    serializer_class = RouteSerializer
+    filter_backends = (filters.OrderingFilter, filters.SearchFilter, DjangoFilterBackend,)
+    search_fields = (
+        'car',
```

Comment:
Does it work with filtering by related objects? How does it know which field to use?